app/dashboard/helpers.py

```diff
@@ -456,6 +456,9 @@ def create_new_bounty(old_bounties, bounty_payload, bounty_details, bounty_id):
             try:
                 issue_kwargs = get_url_dict(new_bounty.github_url)
                 new_bounty.github_issue_details = get_gh_issue_details(**issue_kwargs)
+
+                if latest_old_bounty['is_featured']:
```

Comment:
I think this will send the email every time the bounty changes, not just when its gets featured. See the logic here: https://github.com/gitcoinco/web/blob/d17e80993fdac90e612f570f554f050fda5a2163/app/dashboard/helpers.py#L542

Reply:
yup that might end up getting spammy :/

Reply:
@danlipert @thelostone-mc what do you propose?

Reply:
@iamonuwa Only email the funder when they fund the bounty once 👍

Reply:
fund the bounty once or fund the bounty for the first time and the is_featured flag is turned on?